indexd/default_settings.py

```diff
@@ -32,6 +32,8 @@
     )
 }
 
+CONFIG["PRESIGNED_URL_ENDPT"] = "http://presigned-url-fence-service/"
```

Comment:
I did this because `presigned-url-fence-service` has been separated from `fence-service` so `FENCE_URL` , which would be `http://fence-service`, is different then the `http://presigned-url-fence-service/' which contains the "/data/download". Should I still changed it to `FENCE_URL`?